pkg/conf/conf_test.go

```diff
@@ -181,3 +181,18 @@ func TestFlowStyle(t *testing.T) {
 		}
 	})
 }
+
+func TestIssue271_PollerPanicsWhenExportDoesNotExist(t *testing.T) {
+	path := "../../cmd/tools/doctor/testdata/testConfig.yml"
+	node, _ := LoadConfig(path)
+	poller := node.GetChildS("Pollers").GetChildS("issue-271")
```

Comment:
As going forward replacing node to struct, we would be using LoadHarvestConfig with poller, err := GetPoller2(path, "issue-271")